docs/source/modules/topography.rst

```diff
@@ -0,0 +1,136 @@
+
+The ``idealized_moist_phys`` module is primarily concerned with the land mask. Its job is to pass the information about the land mask to other modules, which in the first instance are the ``mixed_layer`` and the ``surface_flux`` modules, which deal with ocean and surface exchanges with the atmosphere respectively. Both modules need to know where there is ocean/land as so correct physics can be computed in the correct location. In addition to passing this information, ``idealized_moist_phys`` will also use the land mask for some other model features, for example some land based parameters and calculating buckets if using bucket hydrology.
+
+The ``idealized_moist_phys`` namelist options to include the appropriate land mask are:
+
+| ``land_option : 'input'`` (the default is ``'none'``, - this is essentially the on switch)
+| ``land_file_name : 'INPUT/era_land_t42.nc'`` (using the example of the provided land file)
+| ``land_field_name : 'land_mask'`` (this is the default and doesn't need to be specified if that is correct)
+
+``spectral_init_cond``
+^^^^^^^^^^^^^^^^^^^^^^
+Now there is a land mask, topography can be included if desired. If not, this step does can be ignored.
+
+The namelist options below are to include topography from the same file as the land mask. It can be a different file, as long as the grid size is still correct. There are multiple other options for including topography, some of which are unused at the moment.
+
+| ``topography_option : 'input'``  (Tell model to get topography from input file)
+| ``topog_file_name : 'era_land_t42.nc'`` (Again, we use the provided file as the example)
+| ``topog_field_name : 'zsurf'`` (The height field name)
+| ``land_field_name : 'land_mask'`` (The land field name)
+
+NOTE: ``zsurf`` and ``land_mask`` are the default values in Isca and these are the names used in the provided land mask so does not need to be included when using it.
+
+This set up is the standard way to use topography in Isca, using the ``spectral_init_cond`` module. The module deals with topography, setting up the shape of the model boundary layer.
+
+Other ``topography_option`` options are available, but not widely used by the Isca team:
+
+- No topography - the default.
+- Flat topography - the surface geopotential is 0.
+- Interpolated topography - where it will call the ``topography`` module below, however this set up is not currently used.
+- Gaussian Topography - Simple Gaussian-shaped mountains are generated from specified parameters.
+
+2 - Creating Custom Topography
+------------------------------
+
+We provide python code that allows land files to be generated for a range of scenarios. This code is found in ``src/extra/python/isca/land_generator_fn.py``. Some tweaking of the code will be needed to suit the users requirements. The most useful feature of this code is interpolating high resolution data into the correct spectral grid size.
+
```

Comment:
I guess the last sentence ("The most useful feature ... ") belongs to Section 3? I don't think there is any interpolation from high-res landmasks to other grids in land_generator_fn.py